torch/csrc/lazy/core/shape_inference.cpp

```diff
@@ -49,6 +49,10 @@
 
 #include <torch/csrc/lazy/core/shape_inference.h>
 
+#include <torch/csrc/lazy/ts_backend/dynamic_ir.h>
```

Comment:
Please do `lintrunner init && lintrunner -a -r <the_commit_before_your_changes> to fix lint issues above.